test/ui.test.js

```diff
@@ -261,15 +261,15 @@ describe('Test UI: RUM collection when clicked', () => {
       menuItemId: 'openViewDocSource',
     }, tab);
     await aTimeout(1000);
-    expect(logSpy.calledWithMatch('sampleRUM', 'sidekick:context-menu:openViewDocSource')).to.be.true;
+    expect(logSpy.calledWith('sampleRUM', 'sidekick:context-menu:openViewDocSource')).to.be.true;
```

Comment:
Test now includes the location so excluding that from the text since it's not something that we can predictably test.

I.E
http://localhost:2000/?wtr-session-id=RwcYpZfD4QkNBDgyQmnZM